firmware/throttle/src/throttle_control.cpp

```diff
@@ -41,7 +43,10 @@ void check_for_operator_override( void )
         uint32_t accelerator_position_average =
             (accelerator_position.low + accelerator_position.high) / 2;
 
-        if ( accelerator_position_average >= ACCELERATOR_OVERRIDE_THRESHOLD )
+        uint16_t throttle_pedal_override_threshold =
+            oscc_eeprom_read_u16( OSCC_CONFIG_U16_THROTTLE_PEDAL_OVERRIDE_THRESHOLD );
```

Comment:
What are the performance implications for reading the override threshold during every operator override check, vs using a value stored in memory?

Reply:
I don't know, but that's how I was told to do it.